ui/src/main/resources/PhenoTips/SimilarCases.xml

```diff
@@ -1558,7 +1530,6 @@
 }
 .match-details .variants-data .segmented-bar .segmented-unit-fill {
```

Comment:
The widget is usable without external css but css may still be applied to it. Float for example could either be applied in javascript or in css.